cdap-common/src/main/java/io/cdap/cdap/common/metrics/NoOpMetricsCollectionService.java

```diff
@@ -51,6 +51,11 @@ public void gauge(String metricName, long value) {
         // no-op
       }
 
+      @Override
+      public void event(String metricName, long value) {
+
```

Comment:
Add a comment `// no-op` to declare that it is intended to be empty

Reply:
Done.